tests/mixed_int8/test_mixed_int8.py

```diff
@@ -1,689 +0,0 @@
-# coding=utf-8
```

Comment:
We can't remove a whole part of the test suite like this.

Reply:
I changed the name of the original test and added a bnb_kbit folder in tests. Added the missing files.